dotnet3.5/Polly/SynthesizeSpeechExample/SynthesizeSpeech/SynthesizeSpeech.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier:  Apache - 2.0
+
+using Amazon;
+using Amazon.Polly;
+using Amazon.Polly.Model;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace SynthesizeSpeech
+{
+    class SynthesizeSpeech
+    {
+        // The following example text to speech and saves it to an .mp3 file
```

Comment:
"example text to speech" -> "example converts text to speech"?

Reply:
Changed the introduction to:
        // This example uses the Amazon Polly service to convert text to
        // speech. It then saves the converted text to an MP3 file. The
        // code was written against the AWS SDK for .NET 3.5 and .NET 5.0.